### PR TITLE
Enhancement: Footer | SignIn-Card | About section cards | UI improved 

### LINKED ISSUE #369: UI Fixes in [Footer/ Login Form/ About]
### What is this issue related to?

Feature

### Description

To fix and enhance the frontend/ UI changes by re-organizing position of elements in the footer section, login form and about section.

### Expected Behavior

**Footer**
So as to have more height, arrange hyperlinks in single column.

annotely_image

**Login Form**
Forgot password should be below the password text field. 
SignUp option should be centered.

annotely_image (1)
**About Page**
1. Organize Journey, Purpose, Introduction, Offerings into cards.
annotely_image (2)

2. Center "get in touch/ demo" and contact form
annotely_image (3)

### Current Behavior

Issues with:
1. **Footer**- Max height is small
2. **Login Page**- Forgot password not aligned with password input field
3. **About page**- Not having a well-defined layout

### Additional Information (optional)

_No response_

### Confirmation Checklist

- [X] I have searched for similar issues before creating this one.
- [X] I have provided all the necessary information to understand and reproduce the issue.

### PR DESCRIPTION
**Pull request by AryaDharkar**
### Summary
This PR reorganizes several elements across the UI for improved user experience and layout consistency:
1) Footer: Re-arranged the footer elements into a single row for a more streamlined appearance.
2) Signup Form: Moved the "Forgot Password" link directly below the password field, and centered the submit button for better form alignment.
3) About Section: Redesigned the layout by organizing content into cards, making it visually more structured and easier to read.
These changes aim to improve usability and overall visual appeal of the pages.

### Approach

Footer
Arranged hyperlinks in single row.

Login Form
Forgot password below the password text field.
SignUp button centered.

About Page
Organize Journey, Purpose, Introduction, Offerings into cards.

### Visual Updates
FOOTER

https://github.com/user-attachments/assets/33364542-9324-4152-8615-097de050602b

SIGNIN

https://github.com/user-attachments/assets/ca0585b5-0f96-4fdc-9337-66c55033e52f

ABOUT 

https://github.com/user-attachments/assets/c3e33174-0612-4268-ab25-38b2b8c12bab

### How to Test the Changes
The changes are only in UI and CSS files. No testcase edits were done

### Code Coverage Report
Screenshot 2024-09-29 145506

## Checklist

- [x] I have added/updated tests that cover the changes.
- [x] I have updated the documentation to reflect the changes.
- [x] I have passed all test cases (client/server)

### CODE REVIEW

File: client/src/components/footer/Footer.css [hunk 1]
```diff
@@ -13,23 +21,24 @@
 	align-items: center;
 	flex-wrap: wrap;
 	gap: 24px;
-	border-right: var(--border);
+	/* border-right: var(--border); */
```
Comment (DeltaDynamo) [inline client/src/components/footer/Footer.css]: Please remove the comments.

File: client/src/components/footer/Footer.css [hunk 2]
```diff
@@ -2,7 +2,15 @@
...
 	background-color: #f1f1f1;
 	display: flex;
+	flex-direction: column;
 	padding: 12px 0;
+	align-items: center;
+}
+
+.footer-top {
+	width: 100%;
+	display: flex;
```
Comment (DeltaDynamo) [inline client/src/components/footer/Footer.css:12]: Try reducing redundant css properties.
E.g. `footer` is already defined as display: flex, hence the child : `footer-top` will inherit the same.


Also you can try grouping selectors for common properties among various entities.

**Review by aps08 — CHANGES_REQUESTED** [review]
Provide a video demo with web and mobile view. Also work on the comments anubhav has provided.

**Comment by aps08** [comment]
@AryaDharkar

File: client/src/components/footer/Footer.css (hunk 2 above)
Comment (AryaDharkar) [reply client/src/components/footer/Footer.css:12]: Fixed other redundant CSS properties.
In this particular case, the `footer `is `flex-column` and I need `footer-top` in `flex-row`. So not changing that.
Thx for reviewing

File: client/src/components/footer/Footer.css (hunk 1 above)
Comment (AryaDharkar) [reply client/src/components/footer/Footer.css]: Comments removed

**Review by aps08 — APPROVED** [review]

**Pull request merged**